### ISSUE TITLE
Default to no top and footer navigation. 

### ISSUE THREAD

**Issue by nkuehn** [commercetools/commercetools-docs-kit#435]
To allow broader adoption of the kit for new products, less prominently positioned open source offerings and pure experimental / learning use cases I like the idea of allowing to hide the top navigation completely (including the text that opens it) and even defaulting to not showing it.  

 * defaulting to less features active would allow a more natural flow from a new piece of documentation to become part of the main documentation website. It would also allow more fearless experimentation with  the kit. 
 * The change would move the site out of the documentation website context and avoids confusion because there is no way to navigate back into the site in the documentation website. 
 * The footer would also be reduced to showing the very bottom line with company name, privacy policy and imprint only. 
 * The nav would then be activated through a boolean theme option `showTopNavigation`. 
 * The top area would be retained and not hidden to allow for example activating a search functionality or other later features positioned into the top bar. 

Feeback and input appreciated.

**Comment by emmenko** [comment]
Just an idea: make it a theme (core) with most of the current docs theme functionalities, then keep the “docs specific” features in the docs theme.

**Comment by nkuehn** [comment]
I don't think we should scope creep it to a generic website building framework, so "documentation" is still the fundamentally assumed use case.   

This is more about "is it in embedded in the main docs" vs. "a standalone piece of documentation". But documentation in any case. 

So it may rather be "gatsby-theme-docs"  plus an add-on that makes it part of the main documentation mult-microsite-website ("gatsby-theme-multisite-navigation" ?)

**Comment by nkuehn** [comment]
I have  a much simpler idea that would suffice for our ad hoc internal needs:  refactor the component structure a bit so that the elements in the header  bar  (company logo, "Documentation" text, top nav, search) live in own JS module files.  That allows to override them with other or empty components in specific sites by using gatsby theme component shadowing.  That would suffice for most cases.
